overlord/devicestate/devicestate_test.go

```diff
@@ -469,6 +469,38 @@ func (s *deviceMgrSuite) TestDeviceManagerEnsureBootOkSkippedOnNonRunModes(c *C)
 	c.Assert(err, IsNil)
 }
 
+func (s *deviceMgrSuite) switchDevManagerToClassicWithModes(c *C) {
+	release.OnClassic = true
+
+	// mock the modeenv file
+	m := boot.Modeenv{
+		Mode:           "run",
+		RecoverySystem: "20191127",
```

Comment:
this is only set on first boot, so not sure we always want it set for realism. One option is pass it in as needed.

Reply:
Removed